users/models.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth.models import (
+    AbstractBaseUser, BaseUserManager, PermissionsMixin
+)
+from django.db import models
+
+
+class UserManager(BaseUserManager):
+    """Manage user
+    """
+    def create_user(self, email, first_name, last_name, password):
```

Comment:
We need to write flexible code. change how your write codes and stop doing dead end codes.

```
def create_user(self, email, password, **kwargs):
    user = self.model(email=email, is_active=True, **kwargs)
    user.set_password(password)
    user.save()
    
    return user
```

```
def create_superuser(self, email, password, **kwargs):
    return self.create_user(
        email=email,
        password=password,
        is_staff=True,
        is_superuser=True,
        **kwargs,
    )
```